sprint-2/src/Types.ts

```diff
@@ -0,0 +1,14 @@
+export type Person = {
+    readonly id: number;
+    name: string;
+    bio: string;
+    isCreatorOfAda: boolean;
+};
+
+export enum EnumPersonNames {
```

Comment:
Utilizar este Enum na hora de atribuir o nome nas classes para garantir que os nomes sejam do mesmo tipo, (this.name = EnumPersonNames.minhaPessoa)

Reply:
Código corrigido.